src/color/p5.Color.js

```diff
@@ -52,8 +52,18 @@ p5.Color = function(renderer, vals) {
 };
 
 /**
+ * This function returns the color formatted as a string. This can be useful
+ * for debugging, or for using p5.js with other libraries.
  * @method toString
- * @return {String}
+ * @param {Number|String} [format] How the color string will be formatted.
+ * Leaving this empty formats the string as rgba(r, g, b, a).
+ * '#rgb' '#rgba' '#rrggbb' and '#rrggbbaa' format as hexadecimal color codes.
+ * 'hex3' 'hex4' 'hex6' and 'hex8' return the same, without the '#''
+ * 16 'hex' and 'web-color' are aliases of 'hex6'.
```

Comment:
are these ones necessary? I am thinking of a beginner that reads this, it feels a bit overwhelming in the options and text to read through right now.